File: java/pdfclown.lib/src/org/pdfclown/documents/contents/fonts/SymbolEncoding.java
/*

  Copyright 2015 Stefano Chizzolini. http://www.pdfclown.org

  Contributors:
    * Stefano Chizzolini (original code developer, http://www.stefanochizzolini.it)

  This file should be part of the source code distribution of "PDF Clown library"
  (the Program): see the accompanying README files for more info.

  This Program is free software; you can redistribute it and/or modify it under the terms
  of the GNU Lesser General Public License as published by the Free Software Foundation;
  either version 3 of the License, or (at your option) any later version.

  This Program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY,
  either expressed or implied; without even the implied warranty of MERCHANTABILITY or
  FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.

  You should have received a copy of the GNU Lesser General Public License along with this
  Program (see README files); if not, go to the GNU website (http://www.gnu.org/licenses/).

  Redistribution and use, with or without modification, are permitted provided that such
  redistributions retain the above copyright notice, license and disclaimer, along with
  this list of conditions.
*/

package org.pdfclown.documents.contents.fonts;

/**
  Symbol encoding [PDF:1.7:D.4].
  
  @author Stefano Chizzolini (http://www.stefanochizzolini.it)
  @since 0.1.2.1
  @version 0.1.2.1, 04/16/15
*/
final class SymbolEncoding
  extends Encoding
{
  public SymbolEncoding(
    )
  {
    put(0040, "space");
    put(0041, "exclam");
    put(0042, "universal");
    put(0043, "numbersign");
    put(0044, "existential");
    put(0045, "percent");
    put(0046, "ampersand");
    put(0047, "suchthat");
    put(0050, "parenleft");
    put(0051, "parenright");
    put(0052, "asteriskmath");
    put(0053, "plus");
    put(0054, "comma");
    put(0055, "minus");
    put(0056, "period");
    put(0057, "slash");
    put(0060, "zero");
    put(0061, "one");
    put(0062, "two");
    put(0063, "three");
    put(0064, "four");
    put(0065, "five");
    put(0066, "six");
    put(0067, "seven");
    put(0070, "eight");
    put(0071, "nine");
    put(0072, "colon");
    put(0073, "semicolon");
    put(0074, "less");
    put(0075, "equal");
    put(0076, "greater");
    put(0077, "question");
    put(0100, "congruent");
    put(0101,"Alpha");
    put(0102, "Beta");
    put(0103, "Chi");
    put(0104, "Delta");
    put(0105, "Epsilon");
    put(0106, "Phi");
    put(0107, "Gamma");
    put(0110, "Eta");
    put(0111, "Iota");
    put(0112, "theta1");
    put(0113, "Kappa");
    put(0114, "Lambda");
    put(0115, "Mu");
    put(0116, "Nu");
    put(0117, "Omicron");
    put(0120, "Pi");
    put(0121, "Theta");
    put(0122, "Rho");
    put(0123, "Sigma");
    put(0124, "Tau");
    put(0125, "Upsilon");
    put(0126, "sigma1");
    put(0127, "Omega");
    put(0130, "Xi");
    put(0131, "Psi");
    put(0132, "Zeta");
    put(0133, "bracketleft");
    put(0134, "therefore");
    put(0135, "bracketright");
    put(0136, "perpendicular");
    put(0137, "underscore");
    put(0140, "radicalex");
    put(0141, "alpha");
    put(0142, "beta");
    put(0143, "chi");
    put(0144, "delta");
    put(0145, "epsilon");
    put(0146, "phi");
    put(0147, "gamma");
    put(0150, "eta");
    put(0151, "iota");
    put(0152, "phi1");
    put(0153, "kappa");
    put(0154, "lambda");
    put(0155, "mu");
    put(0156, "nu");
    put(0157, "omicron");
    put(0160, "pi");
    put(0161, "theta");
    put(0162, "rho");
    put(0163, "sigma");
    put(0164, "tau");
    put(0165, "upsilon");
    put(0166, "omega1");
    put(0167, "omega");
    put(0170, "xi");
    put(0171, "psi");
    put(0172, "zeta");
    put(0173, "braceleft");
    put(0174, "bar");
    put(0175, "braceright");
    put(0176, "similar");
    put(0240, "Euro");
    put(0241, "Upsilon1");
    put(0242, "minute");
    put(0243, "lessequal");
    put(0244, "fraction");
    put(0245, "infinity");
    put(0246, "florin");
    put(0247, "club");
    put(0250, "diamond");
    put(0251, "heart");
    put(0252, "spade");
    put(0253, "arrowboth");
    put(0254, "arrowleft");
    put(0255, "arrowup");
    put(0256, "arrowright");
    put(0257, "arrowdown");
    put(0260, "degree");
    put(0261, "plusminus");
    put(0262, "second");
    put(0263, "greaterequal");
    put(0264, "multiply");
    put(0265, "proportional");
    put(0266, "partialdiff");
    put(0267, "bullet");
    put(0270, "divide");
    put(0271, "notequal");
    put(0272, "equivalence");
    put(0273, "approxequal");
    put(0274, "ellipsis");
    put(0275, "arrowvertex");
    put(0276, "arrowhorizex");
    put(0277, "carriagereturn");
    put(0300, "aleph");
    put(0301, "Ifraktur");
    put(0302, "Rfraktur");
    put(0303, "weierstrass");
    put(0304, "circlemultiply");
    put(0305, "circleplus");
    put(0306, "emptyset");
    put(0307, "intersection");
    put(0310, "union");
    put(0311, "propersuperset");
    put(0312, "reflexsuperset");
    put(0313, "notsubset");
    put(0314, "propersubset");
    put(0315, "reflexsubset");
    put(0316, "element");
    put(0317, "notelement");
    put(0320, "angle");
    put(0321, "gradient");
    put(0322, "registerserif");
    put(0323, "copyrightserif");
    put(0324, "trademarkserif");
    put(0325, "product");
    put(0326, "radical");
    put(0327, "dotmath");
    put(0330, "logicalnot");
    put(0331, "logicaland");
    put(0332, "logicalor");
    put(0333, "arrowdblboth");
    put(0334, "arrowdblleft");
    put(0335, "arrowdblup");
    put(0336, "arrowdblright");
    put(0337, "arrowdbldown");
    put(0340, "lozenge");
    put(0341, "angleleft");
    put(0342, "registersans");
    put(0343, "copyrightsans");
    put(0344, "trademarksans");
    put(0345, "summation");
    put(0346, "parenlefttp");
    put(0347, "parenleftex");
    put(0350, "parenleftbt");
    put(0351, "bracketlefttp");
    put(0352, "bracketleftex");
    put(0353, "bracketleftbt");
    put(0354, "bracelefttp");
    put(0355, "braceleftmid");
    put(0356, "braceleftbt");
    put(0357, "braceex");
    put(0361, "angleright");
    put(0362, "integral");
    put(0363, "integraltp");
    put(0364, "integralex");
    put(0365, "integralbt");
    put(0366, "parenrighttp");
    put(0367, "parenrightex");
    put(0370, "parenrightbt");
    put(0371, "bracketrighttp");
    put(0372, "bracketrightex");
    put(0373, "bracketrightbt");
    put(0374, "bracerighttp");
    put(0375, "bracerightmid");
    put(0376, "bracerightbt");
  }
}

File: PdfClown-NetStandard/PdfClown/org/pdfclown/documents/contents/fonts/SymbolEncoding.cs
/*
  Copyright 2015 Stefano Chizzolini. http://www.pdfclown.org

  Contributors:
    * Stefano Chizzolini (original code developer, http://www.stefanochizzolini.it)

  This file should be part of the source code distribution of "PDF Clown library" (the
  Program): see the accompanying README files for more info.

  This Program is free software; you can redistribute it and/or modify it under the terms
  of the GNU Lesser General Public License as published by the Free Software Foundation;
  either version 3 of the License, or (at your option) any later version.

  This Program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY,
  either expressed or implied; without even the implied warranty of MERCHANTABILITY or
  FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.

  You should have received a copy of the GNU Lesser General Public License along with this
  Program (see README files); if not, go to the GNU website (http://www.gnu.org/licenses/).

  Redistribution and use, with or without modification, are permitted provided that such
  redistributions retain the above copyright notice, license and disclaimer, along with
  this list of conditions.
*/

namespace org.pdfclown.documents.contents.fonts
{
  /**
    <summary>Symbol encoding [PDF:1.7:D.4].</summary>
  */
  internal sealed class SymbolEncoding
    : Encoding
  {
    public SymbolEncoding(
      )
    {
      Put(32, "space");
      Put(33, "exclam");
      Put(34, "universal");
      Put(35, "numbersign");
      Put(36, "existential");
      Put(37, "percent");
      Put(38, "ampersand");
      Put(39, "suchthat");
      Put(40, "parenleft");
      Put(41, "parenright");
      Put(42, "asteriskmath");
      Put(43, "plus");
      Put(44, "comma");
      Put(45, "minus");
      Put(46, "period");
      Put(47, "slash");
      Put(48, "zero");
      Put(49, "one");
      Put(50, "two");
      Put(51, "three");
      Put(52, "four");
      Put(53, "five");
      Put(54, "six");
      Put(55, "seven");
      Put(56, "eight");
      Put(57, "nine");
      Put(58, "colon");
      Put(59, "semicolon");
      Put(60, "less");
      Put(61, "equal");
      Put(62, "greater");
      Put(63, "question");
      Put(64, "congruent");
      Put(65,"Alpha");
      Put(66, "Beta");
      Put(67, "Chi");
      Put(68, "Delta");
      Put(69, "Epsilon");
      Put(70, "Phi");
      Put(71, "Gamma");
      Put(72, "Eta");
      Put(73, "Iota");
      Put(74, "theta1");
      Put(75, "Kappa");
      Put(76, "Lambda");
      Put(77, "Mu");
      Put(78, "Nu");
      Put(79, "Omicron");
      Put(80, "Pi");
      Put(81, "Theta");
      Put(82, "Rho");
      Put(83, "Sigma");
      Put(84, "Tau");
      Put(85, "Upsilon");
      Put(86, "sigma1");
      Put(87, "Omega");
      Put(88, "Xi");
      Put(89, "Psi");
      Put(90, "Zeta");
      Put(91, "bracketleft");
      Put(92, "therefore");
      Put(93, "bracketright");
      Put(94, "perpendicular");
      Put(95, "underscore");
      Put(96, "radicalex");
      Put(97, "alpha");
      Put(98, "beta");
      Put(99, "chi");
      Put(100, "delta");
      Put(101, "epsilon");
      Put(102, "phi");
      Put(103, "gamma");
      Put(104, "eta");
      Put(105, "iota");
      Put(106, "phi1");
      Put(107, "kappa");
      Put(108, "lambda");
      Put(109, "mu");
      Put(110, "nu");
      Put(111, "omicron");
      Put(112, "pi");
      Put(113, "theta");
      Put(114, "rho");
      Put(115, "sigma");
      Put(116, "tau");
      Put(117, "upsilon");
      Put(118, "omega1");
      Put(119, "omega");
      Put(120, "xi");
      Put(121, "psi");
      Put(122, "zeta");
      Put(123, "braceleft");
      Put(124, "bar");
      Put(125, "braceright");
      Put(126, "similar");
      Put(160, "Euro");
      Put(161, "Upsilon1");
      Put(162, "minute");
      Put(163, "lessequal");
      Put(164, "fraction");
      Put(165, "infinity");
      Put(166, "florin");
      Put(167, "club");
      Put(168, "diamond");
      Put(169, "heart");
      Put(170, "spade");
      Put(171, "arrowboth");
      Put(172, "arrowleft");
      Put(173, "arrowup");
      Put(174, "arrowright");
      Put(175, "arrowdown");
      Put(176, "degree");
      Put(177, "plusminus");
      Put(178, "second");
      Put(179, "greaterequal");
      Put(180, "multiply");
      Put(181, "proportional");
      Put(182, "partialdiff");
      Put(183, "bullet");
      Put(184, "divide");
      Put(185, "notequal");
      Put(186, "equivalence");
      Put(187, "approxequal");
      Put(188, "ellipsis");
      Put(189, "arrowvertex");
      Put(190, "arrowhorizex");
      Put(191, "carriagereturn");
      Put(192, "aleph");
      Put(193, "Ifraktur");
      Put(194, "Rfraktur");
      Put(195, "weierstrass");
      Put(196, "circlemultiply");
      Put(197, "circleplus");
      Put(198, "emptyset");
      Put(199, "intersection");
      Put(200, "union");
      Put(201, "propersuperset");
      Put(202, "reflexsuperset");
      Put(203, "notsubset");
      Put(204, "propersubset");
      Put(205, "reflexsubset");
      Put(206, "element");
      Put(207, "notelement");
      Put(208, "angle");
      Put(209, "gradient");
      Put(210, "registerserif");
      Put(211, "copyrightserif");
      Put(212, "trademarkserif");
      Put(213, "product");
      Put(214, "radical");
      Put(215, "dotmath");
      Put(216, "logicalnot");
      Put(217, "logicaland");
      Put(218, "logicalor");
      Put(219, "arrowdblboth");
      Put(220, "arrowdblleft");
      Put(221, "arrowdblup");
      Put(222, "arrowdblright");
      Put(223, "arrowdbldown");
      Put(224, "lozenge");
      Put(225, "angleleft");
      Put(226, "registersans");
      Put(227, "copyrightsans");
      Put(228, "trademarksans");
      Put(229, "summation");
      Put(230, "parenlefttp");
      Put(231, "parenleftex");
      Put(232, "parenleftbt");
      Put(233, "bracketlefttp");
      Put(234, "bracketleftex");
      Put(235, "bracketleftbt");
      Put(236, "bracelefttp");
      Put(237, "braceleftmid");
      Put(238, "braceleftbt");
      Put(239, "braceex");
      Put(241, "angleright");
      Put(242, "integral");
      Put(243, "integraltp");
      Put(244, "integralex");
      Put(245, "integralbt");
      Put(246, "parenrighttp");
      Put(247, "parenrightex");
      Put(248, "parenrightbt");
      Put(249, "bracketrighttp");
      Put(250, "bracketrightex");
      Put(251, "bracketrightbt");
      Put(252, "bracerighttp");
      Put(253, "bracerightmid");
      Put(254, "bracerightbt");
    }
  }
}